### ISSUE TITLE
Extension landing pages need an accessibility pass 

### ISSUE THREAD

**Issue by technoboy10** [LLK/scratch-www#1991]
The extension landing pages landed in #1981 have a number of accessibility issues (missing alt/titles, issues with buttons, links, images, etc.). We should do an accessibility pass to resolve these issues on those pages.

**Comment by LiFaytheGoblin** [comment]
ev3
- [ ] add alts to images

microbit
- [ ] add alts to images

wedo
- [ ] add alts to images

**Comment by LiFaytheGoblin** [comment]
http://wave.webaim.org/report#/https://scratch.mit.edu/ev3

**Comment by LiFaytheGoblin** [comment]
DISCUSSION NEEDED

- [ ] add hidden label to search icon
- [ ] add alts to icons and decorative pictures that don't add anything new to the text
- [ ] can the document language be identified to the user via <html lang=....>?
- [ ] add hidden h1 

@chrisgarrity  what do you think

**Comment by LiFaytheGoblin** [comment]
Working on this at https://github.com/LiFaytheGoblin/scratch-www/tree/1991/accessibility-extension-landing

**Comment by chrisgarrity** [comment]
@LiFaytheGoblin 

- [ ] add hidden label to search icon: this should work. However, it's part of the nav bar so it'll be changing it for every page.
- [ ] add alts to icons and decorative pictures that don't add anything new to the text: Definitely, often we use the same text for the `alt` and `title`. So it may not even need new strings.
- [ ] can the document language be identified to the user via <html lang=....>?: The HTML element is part of the template for all pages, not just the extension landing pages. We'd probably need to add something to [init.js](https://github.com/LLK/scratch-www/blob/develop/src/init.js) that adds a `lang` attribute to the HTML element once we know what the current language is.
- [ ] add hidden h1: Could the main header for the landing page be an H1 instead of H2? (i.e. the LEGO MINDSTORMS EV3 header) - it would mean overriding the style for `.extension-header h1` instead of `.extension-header h2`

**Comment by chrisgarrity** [comment]
We should be able to set the lang attribute on the root element with 
```js
document.documentElement.lang = <locale determined by init.js>
```

**Comment by LiFaytheGoblin** [comment]
Thanks @chrisgarrity! I will look into the things you suggested!

**Comment by LiFaytheGoblin** [comment]
After reading a bit more on this on https://www.w3.org/WAI/tutorials/images/decorative/ the accessibility friendliest way would be to add empty alt tags so I'm doing this now.

**Comment by LiFaytheGoblin** [comment]
Um, I can't find the root element ._. @chrisgarrity Maybe you could point me to it?

**Comment by LiFaytheGoblin** [comment]
adding document.documentElement.lang = <locale determined by init.js> to init js did the trick :)
